command-derive/command-derive-impl/src/lib.rs

```diff
@@ -51,14 +34,21 @@ pub fn command_derive(input: TokenStream) -> TokenStream {
             .collect();
         if metas
             .iter()
-            .any(|meta| VariantSkip::from_meta(meta).is_ok())
+            .any(|meta| CommandMarker("skip").validate(meta).is_ok())
         {
-            continue;
+            return str_as_compile_error("#[command(skip)] was hard-deprecated, sorry");
```

Comment:
Why skip was hard deprecated?

Reply:
Because we don't need it anymore. It was easier to make than `#[command(flatten)]`, and I've put it on variants with nested structs in previous pull, so the macro would be usable.